include/magma/vk/commandBuffer.h

```diff
@@ -10,9 +10,14 @@ class CommandBufferArr
 public:
     CommandBufferArr() = default;
     CommandBufferArr(VkDevice device, VkCommandPool commandPool, uint32_t count);
+    ~CommandBufferArr();
+
     void allocate(VkDevice device, VkCommandPool commandPool, uint32_t count);
     VkCommandBuffer beginCmdBuf(uint32_t i);
     void endCmdBuf(uint32_t i);
+    void endAndSubmitCmdBuf(uint32_t i, VkQueue queue);
+    void resetCmdBuf(uint32_t i, VkCommandBufferResetFlags flag=VK_NULL_HANDLE);
+    void freeCmdBuf(VkDevice device, VkCommandPool commandPool);
```

Comment:
freeCmdBuf is error prone method, because you need to pass commandPool as an argument.
It's better to put commandPool to a class member, when you allocate command buffers.

Reply:
this method may be change in future commits with all commandBuffer.h

Reply:
ok